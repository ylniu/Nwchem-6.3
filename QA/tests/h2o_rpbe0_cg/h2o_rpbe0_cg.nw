echo
start h2o_rpbe0_cg

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end

basis
  O library 6-31g*
  H library 6-31g*
end

dft
  cgmin
  xc pbe0
end

task dft
