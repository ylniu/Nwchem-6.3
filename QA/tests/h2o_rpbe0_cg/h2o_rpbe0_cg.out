 argument  1 = h2o_rpbe0_cg.nw
 


============================== echo of input deck ==============================
echo
start h2o_rpbe0_cg

memory global 40 mb stack 23 mb heap 5 mb

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end

basis
  O library 6-31g*
  H library 6-31g*
end

dft
  cgmin
  xc pbe0
end

task dft
================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = orion
    program       = ../../../bin/LINUX64/nwchem
    date          = Wed Jan 12 10:56:34 2011

    compiled      = Wed_Jan_12_10:20:28_2011
    source        = /home/niri/nwchem/nwchem
    nwchem branch = Development
    input         = h2o_rpbe0_cg.nw
    prefix        = h2o_rpbe0_cg.
    data base     = ./h2o_rpbe0_cg.db
    status        = startup
    nproc         =        4
    time left     =     -1s



           Memory information
           ------------------

    heap     =     655361 doubles =      5.0 Mbytes
    stack    =    3014657 doubles =     23.0 Mbytes
    global   =    5242880 doubles =     40.0 Mbytes (distinct from heap & stack)
    total    =    8912898 doubles =     68.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 
 C2V symmetry detected

          ------
          auto-z
          ------
 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.22140000
    2 H                    1.0000    -1.43000000     0.00000000    -0.88560000
    3 H                    1.0000     1.43000000     0.00000000    -0.88560000
 
      Atomic Mass 
      ----------- 
 
      O                 15.994910
      H                  1.007825
 

 Effective nuclear repulsion energy (a.u.)       9.1971984402

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
      Symmetry information
      --------------------
 
 Group name             C2v       
 Group number             16
 Group order               4
 No. of unique centers     2
 
      Symmetry unique atoms
 
     1    2
 


                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles
 
      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95697
    2 Stretch                  1     3                       0.95697
    3 Bend                     2     1     3               104.51124
 
 
            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000     0.11715984
 H                    -0.75672347     0.00000000    -0.46863937
 H                     0.75672347     0.00000000    -0.46863937
 
 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80841  |     1.80841
    3 H                |   1 O                |     1.80841  |     1.80841
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.51
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem/src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521
 
  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767
 
  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159
 
  4 S  2.70005800E-01  1.000000
 
  5 P  2.70005800E-01  1.000000
 
  6 D  8.00000000E-01  1.000000
 
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757
 
  2 S  1.61277800E-01  1.000000
 


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31g*                  6       15   3s2p1d
 H                           6-31g*                  2        2   2s


 
                                 NWChem DFT Module
                                 -----------------
 
 
  Caching 1-el integrals 
 
            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     3
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:    19
                     number of shells:    10
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04
 
              XC Information
              --------------
                         PBE0 Method XC Functional
                     Hartree-Fock (Exact) Exchange  0.250          
          PerdewBurkeErnzerhof Exchange Functional  0.750          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local
 
             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       49           5.0       434
          H                   0.35       45           6.0       434
          Grid pruning is: on 
          Number of quadrature shells:    94
          Spatial weights used:  Erf1
 
          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

 
      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -75.75081731
 
      Non-variational initial energy
      ------------------------------

 Total energy =     -75.919952
 1-e energy   =    -121.737767
 2-e energy   =      36.620616
 HOMO         =      -0.470482
 LUMO         =       0.114886
 
 
      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------
 
  Numbering of irreducible representations: 
 
     1 a1          2 a2          3 b1          4 b2      
 
  Orbital symmetries:
 
     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1          8 b1          9 a1         10 b2      
    11 a1         12 b1         13 a1         14 a1         15 a2      
 


 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          5.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


 #quartets = 1.009D+03 #integrals = 5.756D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2o_rpbe0_cg.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  50970
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Grid_pts file          = ./h2o_rpbe0_cg.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      5        Max. recs in file   =    271825


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -76.2884883529  8.79D-01  3.55D-01      0.1
                 2      -76.3223934498  2.49D-01  1.11D-01      0.1
                 3      -76.3254661741  4.84D-02  2.17D-02      0.1
                 4      -76.3255592201  4.90D-04  2.22D-04      0.2


         Total DFT energy =      -76.325559220096
      One electron energy =     -123.106950476515
           Coulomb energy =       44.614475023951
    Exchange-Corr. energy =       -7.030282207730
 Nuclear repulsion energy =        9.197198440198

 Numeric. integr. density =       10.000001188127

     Total iterative time =      0.1s


 
                       DFT Final Molecular Orbital Analysis
                       ------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-1.028129D+00  Symmetry=a1
              MO Center=  1.5D-16,  4.2D-17, -8.7D-02, r^2= 5.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.468643  1 O  s                  6      0.433719  1 O  s          
     1     -0.208749  1 O  s          
 
 Vector    3  Occ=2.000000D+00  E=-5.361584D-01  Symmetry=b1
              MO Center= -7.6D-17, -3.0D-17, -1.1D-01, r^2= 7.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.516942  1 O  px                 7      0.277124  1 O  px         
    16     -0.238122  2 H  s                 18      0.238122  3 H  s          
 
 Vector    4  Occ=2.000000D+00  E=-3.853450D-01  Symmetry=a1
              MO Center=  3.9D-16, -2.6D-16,  2.0D-01, r^2= 6.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.553169  1 O  pz                 6      0.387245  1 O  s          
     9      0.381912  1 O  pz                 2      0.178375  1 O  s          
 
 Vector    5  Occ=2.000000D+00  E=-3.049298D-01  Symmetry=b2
              MO Center= -1.1D-17,  5.9D-17,  9.6D-02, r^2= 6.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.640866  1 O  py                 8      0.510018  1 O  py         
 
 Vector    6  Occ=0.000000D+00  E= 8.425280D-02  Symmetry=a1
              MO Center= -7.8D-16,  2.4D-17, -6.2D-01, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.283596  1 O  s                 17     -0.982028  2 H  s          
    19     -0.982028  3 H  s                  9     -0.474365  1 O  pz         
     5     -0.268944  1 O  pz         
 
 Vector    7  Occ=0.000000D+00  E= 1.705050D-01  Symmetry=b1
              MO Center= -1.1D-16, -2.0D-20, -5.7D-01, r^2= 2.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      1.305202  2 H  s                 19     -1.305202  3 H  s          
     7      0.772840  1 O  px                 3      0.403331  1 O  px         
 
 Vector    8  Occ=0.000000D+00  E= 8.010988D-01  Symmetry=b1
              MO Center=  3.1D-16,  1.2D-32, -1.2D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      0.837578  2 H  s                 18     -0.837578  3 H  s          
    19      0.611191  3 H  s                 17     -0.611191  2 H  s          
     7      0.487289  1 O  px                12      0.351018  1 O  dxz        
     3      0.158023  1 O  px         
 
 Vector    9  Occ=0.000000D+00  E= 8.955854D-01  Symmetry=a1
              MO Center= -2.6D-16,  2.4D-17, -1.8D-01, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.718515  1 O  pz                16      0.703639  2 H  s          
    18      0.703639  3 H  s                 19     -0.521835  3 H  s          
    17     -0.521835  2 H  s                  9     -0.381256  1 O  pz         
    10      0.175180  1 O  dxx        
 
 Vector   10  Occ=0.000000D+00  E= 9.167664D-01  Symmetry=a1
              MO Center=  3.2D-16, -5.6D-18,  2.9D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.373261  1 O  s                  9      0.833054  1 O  pz         
     2     -0.822399  1 O  s                  5     -0.547625  1 O  pz         
    18      0.487688  3 H  s                 16      0.487688  2 H  s          
    17     -0.377665  2 H  s                 19     -0.377665  3 H  s          
    13     -0.329053  1 O  dyy               15     -0.277920  1 O  dzz        
 
 Vector   11  Occ=0.000000D+00  E= 9.174344D-01  Symmetry=b2
              MO Center=  1.0D-16,  3.1D-16,  1.1D-01, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.036633  1 O  py                 4     -0.961920  1 O  py         
 
 Vector   12  Occ=0.000000D+00  E= 1.097155D+00  Symmetry=b1
              MO Center=  1.2D-16,  4.5D-21,  8.3D-02, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.630301  1 O  px                 3     -0.998128  1 O  px         
    19     -0.955445  3 H  s                 17      0.955445  2 H  s          
 
 Vector   13  Occ=0.000000D+00  E= 1.209227D+00  Symmetry=a1
              MO Center= -3.3D-16, -1.7D-16, -3.7D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.650684  1 O  s                  2     -1.474535  1 O  s          
     9     -1.094882  1 O  pz                17     -0.803491  2 H  s          
    19     -0.803491  3 H  s                 10     -0.660810  1 O  dxx        
     5      0.405687  1 O  pz                15     -0.388412  1 O  dzz        
    18     -0.328017  3 H  s                 16     -0.328017  2 H  s          
 
 Vector   14  Occ=0.000000D+00  E= 1.737101D+00  Symmetry=a1
              MO Center=  1.7D-18,  1.5D-17,  1.7D-01, r^2= 6.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.010234  1 O  dzz               10     -0.578651  1 O  dxx        
    13     -0.361508  1 O  dyy                6     -0.269089  1 O  s          
     9      0.150146  1 O  pz         
 
 Vector   15  Occ=0.000000D+00  E= 1.751372D+00  Symmetry=a2
              MO Center= -3.0D-16,  3.1D-17,  1.2D-01, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.732051  1 O  dxy        
 

 Parallel integral file used       4 records with       0 large values


 Task  times  cpu:        0.2s     wall:        1.5s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  275      275     1521      640     1337        9        0        0     
number of processes/call 1.41e+00 1.32e+00 1.18e+00 2.00e+00 0.00e+00
bytes total:             9.89e+05 4.47e+05 3.73e+05 7.20e+03 0.00e+00 0.00e+00
bytes remote:            2.09e+05 4.73e+04 1.25e+05 -3.00e+03 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 33960 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        22	        52
	current total bytes		         0	         0
	maximum total bytes		   1552424	  22509584
	maximum total K-bytes		      1553	     22510
	maximum total M-bytes		         2	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        0.2s     wall:        1.7s
